aqi/algos/caqi.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+from decimal import *
+
+from aqi.constants import (POLLUTANT_PM25, POLLUTANT_PM10,
+                           POLLUTANT_O3_1H, POLLUTANT_CO_8H, POLLUTANT_SO2_1H,
+                           POLLUTANT_NO2_1H)
+from aqi.algos.base import PiecewiseAQI
+
+
+class AQI(PiecewiseAQI):
+    """Implementation of the EPA AQI algorithm.
+    """
+
+    piecewise = {
+        'aqi': [
+            (0, 25),
+            (26, 50),
+            (51, 75),
+            (76, 100),
+            (101, 1000)],
```

Comment:
I should probably think about adding code to handle open scales like this one. Let me know if you have suggestions here.

Reply:
No suggestions yet:)
Interesting thing is that I did not find algorithm for calculating CAQI in https://www.airqualitynow.eu/download/CITEAIR-Comparing_Urban_Air_Quality_across_Borders.pdf
But I checked output of Your algorithm and it matches official measurement in my city.